# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1468330809 14400
#      Tue Jul 12 09:40:09 2016 -0400
# Node ID b8f92ea626e74859607912766852c3d71de77c5d
# Parent  6119019d42f65acab2becf94c06b8c04ea7b6725
include builtins.h only where needed

* graphics.cc, sysdep.cc, __init_gnuplot__.cc, ov-usr-fcn.cc: Don't
include builtins.h.

diff --git a/libinterp/corefcn/graphics.cc b/libinterp/corefcn/graphics.cc
--- a/libinterp/corefcn/graphics.cc
+++ b/libinterp/corefcn/graphics.cc
@@ -37,17 +37,16 @@ along with Octave; see the file COPYING.
 
 #include "cmd-edit.h"
 #include "file-ops.h"
 #include "file-stat.h"
 #include "oct-locbuf.h"
 #include "oct-time.h"
 #include "singleton-cleanup.h"
 
-#include "builtins.h"
 #include "defun.h"
 #include "display.h"
 #include "error.h"
 #include "graphics.h"
 #include "input.h"
 #include "interpreter.h"
 #include "ov.h"
 #include "ovl.h"
diff --git a/libinterp/corefcn/sysdep.cc b/libinterp/corefcn/sysdep.cc
--- a/libinterp/corefcn/sysdep.cc
+++ b/libinterp/corefcn/sysdep.cc
@@ -67,17 +67,16 @@ along with Octave; see the file COPYING.
 #include "lo-math.h"
 #include "mach-info.h"
 #include "oct-env.h"
 #include "quit.h"
 #include "unistd-wrappers.h"
 #include "unsetenv-wrapper.h"
 
 #include "Cell.h"
-#include "builtins.h"
 #include "defun.h"
 #include "display.h"
 #include "error.h"
 #include "errwarn.h"
 #include "input.h"
 #include "octave.h"
 #include "ov.h"
 #include "ovl.h"
diff --git a/libinterp/dldfcn/__init_gnuplot__.cc b/libinterp/dldfcn/__init_gnuplot__.cc
--- a/libinterp/dldfcn/__init_gnuplot__.cc
+++ b/libinterp/dldfcn/__init_gnuplot__.cc
@@ -29,17 +29,16 @@ To initialize:
 
 */
 
 #if defined (HAVE_CONFIG_H)
 #  include "config.h"
 #endif
 
 #include "build-env.h"
-#include "builtins.h"
 #include "defun-dld.h"
 #include "error.h"
 #include "file-stat.h"
 #include "graphics.h"
 #include "oct-env.h"
 #include "parse.h"
 #include "utils.h"
 #include "variables.h"
diff --git a/libinterp/octave-value/ov-usr-fcn.cc b/libinterp/octave-value/ov-usr-fcn.cc
--- a/libinterp/octave-value/ov-usr-fcn.cc
+++ b/libinterp/octave-value/ov-usr-fcn.cc
@@ -26,17 +26,16 @@ along with Octave; see the file COPYING.
 
 #include <sstream>
 
 #include "str-vec.h"
 
 #include "call-stack.h"
 #include <defaults.h>
 #include "Cell.h"
-#include "builtins.h"
 #include "defun.h"
 #include "error.h"
 #include "errwarn.h"
 #include "input.h"
 #include "ovl.h"
 #include "ov-usr-fcn.h"
 #include "ov.h"
 #include "pager.h"
